### ISSUE TITLE
[bug] bpython don't support non-ascii char in sys.ps1

### ISSUE THREAD

**Issue by Freed-Wu** [bpython/bpython#911]
```python
import sys
sys.ps1 = ''
```

In python, it works.
![Screenshot from 2021-09-18 21-24-59](https://user-images.githubusercontent.com/32936898/133890455-ca6f2488-51c1-495b-a29f-4c548ab50a63.png)

In bpython, it brokes.

```
Traceback (most recent call last):
  File "/usr/bin/bpython", line 33, in <module>
    sys.exit(load_entry_point('bpython==0.21', 'console_scripts', 'bpython')())
  File "/usr/lib/python3.9/site-packages/bpython/curtsies.py", line 201, in main
    exit_value = repl.mainloop(True, paste)
  File "/usr/lib/python3.9/site-packages/bpython/curtsies.py", line 106, in mainloop
    self.process_event(bpythonevents.RunStartupFileEvent())
  File "/usr/lib/python3.9/site-packages/bpython/curtsiesfrontend/repl.py", line 611, in process_event
    return self.process_control_event(e)
  File "/usr/lib/python3.9/site-packages/bpython/curtsiesfrontend/repl.py", line 662, in process_control_event
    self.startup()
  File "/usr/lib/python3.9/site-packages/bpython/repl.py", line 480, in startup
    source = f.read()
  File "/usr/lib/python3.9/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xee in position 242: ordinal not in range(128)
```

**Comment by Freed-Wu** [comment]
any non-ascii char in `$PYTHONSTARTUP` will result in this bug, not only sys.ps1

**Comment by sebastinas** [comment]
This was fixed in 2f4e69a92595321e6ab17ae8e24ddc1f6691e9f4.
